packages/components/popover/stories/popover.stories.tsx

```diff
@@ -453,6 +458,15 @@ export const DisableAnimation = {
   },
 };
 
+export const ShouldNotCloseOnScroll = {
```

Comment:
This story is not necessary since you added it on control. Users can just try on any stories.